bin/verify.sh

```diff
@@ -21,60 +21,54 @@ loadValuesFromFile $ADDRESSES_FILE
 
 [[ -z "$GOVERNANCE" ]] && GOVERNANCE="$ETH_FROM"
 
-message verify root $ROOT_CONTRACT
-dapp verify-contract --async 'src/root.sol:TinlakeRoot' $ROOT_CONTRACT "$ETH_FROM" "$GOVERNANCE"
+message Verify root $ROOT_CONTRACT
+dapp verify-contract --async 'src/root.sol:TinlakeRoot' $ROOT_CONTRACT "$ETH_FROM" "$GOVERNANCE" || true
 
+message Verify borrower deployer $BORROWER_DEPLOYER
+dapp verify-contract --async "src/borrower/deployer.sol:BorrowerDeployer" $BORROWER_DEPLOYER $ROOT_CONTRACT $TITLE_FAB $SHELF_FAB $PILE_FAB $FEED_FAB $TINLAKE_CURRENCY '"Tinlake Loan Token"' '"TLNFT"' $DISCOUNT_RATE || true
 
-message verify borrower contracts
-message verify borrower deployer $BORROWER_DEPLOYER
-dapp verify-contract --async "src/borrower/deployer.sol:BorrowerDeployer" $BORROWER_DEPLOYER $ROOT_CONTRACT $TITLE_FAB $SHELF_FAB $PILE_FAB $COLLECTOR_FAB $FEED_FAB $TINLAKE_CURRENCY '"Tinlake Loan Token"' '"TLNFT"' $DISCOUNT_RATE
+message Verify title $TITLE 
+dapp verify-contract --async 'lib/tinlake-title/src/title.sol:Title' $TITLE '"Tinlake Loan Token"' '"TLNFT"' || true
 
-message verify title $TITLE 
-dapp verify-contract --async 'lib/tinlake-title/src/title.sol:Title' $TITLE '"Tinlake Loan Token"' '"TLNFT"'
+message Verify pile $PILE
+dapp verify-contract --async 'src/borrower/pile.sol:Pile' $PILE || true
```

Comment:
Why do we need that `|| true`?  Asking out of curiosity

Reply:
If the contract was already verified, this prevents the verification script from exiting, and it will just continue with the rest. I regularly had issues where it would somehow fail mid-script and I had to comment out certain parts, this way you can just re-run it.